gradle.properties

```diff
@@ -16,7 +16,7 @@
 group=org.apache.kafka
 # NOTE: When you change this version number, you should also make sure to update
 # the version numbers in tests/kafkatest/__init__.py and kafka-merge-pr.py.
-version=1.0.3-SNAPSHOT
+version=1.0.3-flipp
```

Comment:
We're going to be maintaining this in-house in jfrog, until I can get this accepted into core Kafka (by handling the TODOs, further optimizations and having that pass the kafka code review)